### ISSUE TITLE
Path Based Enviroment Variable

### ISSUE THREAD

**Issue by NilsBaumgartner1994** [ohmyform/ohmyform#178]
#### About the Feature

Since with docker-compose all requests are routed to the topleve a path based subdir variable would be nice.

Service running at: example.com/ohmyform/form

Frontend currently tries to fetch from: example.com/_next/static.....
Wanted behaviour: fetch from example.com/ohymyform/form/_next/static.....

**Comment by wodka** [comment]
good point. We should then update this in https://github.com/ohmyform/ui/blob/master/doc/environment.md and add through https://nextjs.org/docs/api-reference/next.config.js/basepath

**Comment by NilsBaumgartner1994** [comment]
Thanks. It would help (us) to setup multiple instance of ohmyform given a subpath.
